src/org/usfirst/frc/team694/robot/commands/CrabArmDeacquireCommand.java

```diff
@@ -14,7 +14,7 @@ protected void initialize() {
     }
 
     protected void execute() {
-        Robot.crabArm.deacquire();
+        Robot.crabArm.flapOut();
```

Comment:
Flapping out should probably not be running at full speed, but the name change is welcome.